### PR TITLE
Add LANGUAGE_COOKIE_* settings

### PR DESCRIPTION
**Pull request by andersk**
https://docs.djangoproject.com/en/4.1/ref/settings/#language-cookie-httponly
https://docs.djangoproject.com/en/4.1/ref/settings/#language-cookie-samesite

### CODE REVIEW

File: django-stubs/conf/global_settings.pyi [hunk 1]
```diff
@@ -61,6 +61,8 @@ LANGUAGE_COOKIE_NAME: str = ...
 LANGUAGE_COOKIE_AGE: Optional[int] = ...
 LANGUAGE_COOKIE_DOMAIN: Optional[str] = ...
 LANGUAGE_COOKIE_PATH: str = ...
+LANGUAGE_COOKIE_HTTPONLY: bool = ...
+LANGUAGE_COOKIE_SAMESITE: Optional[Literal["Lax", "Strict", "None", False]] = ...
```
Comment (adamchainz) [inline django-stubs/conf/global_settings.pyi]: Why make the samesite settings `Optional`?

`None` is not documented as an allowed value:

https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-SESSION_COOKIE_SAMESITE

Comment (andersk) [reply django-stubs/conf/global_settings.pyi]: `None` is the default for `LANGUAGE_COOKIE_SAMESITE`. The rest is for consistency. Django itself gives the same meaning to `False` and `None` (not to be confused with `"None"`), as far as I can tell.

Comment (flaeppe) [reply django-stubs/conf/global_settings.pyi]: The previous change I made for some of these was to remove `Optional`, to align with the documentation: https://github.com/typeddjango/django-stubs/pull/1110

I'm completely guessing right now, but might `None` be due to backwards compatibility?

It feels like what is documented should take precedence here. At least I would like to get an error running Mypy if I'm using `None` and instead should use e.g. `False` to disable, which the docs state.

Comment (adamchainz) [reply django-stubs/conf/global_settings.pyi]: Yeah I also think we should go with what is documented, as that’s all Django promises to support. Yes `None` happens to work, and that’s unlikely to change, but `False` is all that is guaranteed.

File: django-stubs/conf/global_settings.pyi [hunk 2]
```diff
@@ -61,6 +61,8 @@ LANGUAGE_COOKIE_NAME: str = ...
 LANGUAGE_COOKIE_AGE: Optional[int] = ...
 LANGUAGE_COOKIE_DOMAIN: Optional[str] = ...
 LANGUAGE_COOKIE_PATH: str = ...
+LANGUAGE_COOKIE_HTTPONLY: bool = ...
+LANGUAGE_COOKIE_SAMESITE: Literal["Lax", "Strict", "None", False] = ...
```
Comment (andersk) [inline django-stubs/conf/global_settings.pyi:65]: This is definitely wrong—the default value *is* `None` ([documentation](https://docs.djangoproject.com/en/4.1/ref/settings/#language-cookie-samesite), [code](https://github.com/django/django/blob/4.1.3/django/conf/global_settings.py#L171)), so you *can* read `settings.LANGUAGE_COOKIE_SAMESITE` and get `None`, so this annotation breaks type safety.

Comment (andersk) [reply django-stubs/conf/global_settings.pyi:65]: - #1246

**Comment by sshishov** [comment]
`LANGUAGE_COOKIE_SECURE` is missing, guys

**Comment by sobolevn** [comment]
PR is welcome!

**Pull request merged**